MinersLegacy/XmrigCryptonight-v5.5.3-NVIDIA.ps1

```diff
@@ -27,18 +27,10 @@ if ($CUDAVersion -and [System.Version]$CUDAVersion -lt [System.Version]$Required
     Write-Log -Level Warn "Miner ($($Name)) requires CUDA version $($RequiredCUDAVersion) or above (installed version is $($CUDAVersion)). Please update your Nvidia drivers. "
     return
  }
-
-if ($CUDAVersion -lt [System.Version]"10.0.0") { 
-    $HashSHA256 = "218C5F0B3337C76835E2E5603BD804983A344F88FFFF937BF222F2B16C4F52C9"
-    $Uri = "https://github.com/xmrig/xmrig-nvidia/releases/download/v2.14.5/xmrig-nvidia-2.14.5-cuda9_2-win64.zip"
- }
-elseif ($CUDAVersion -lt [System.Version]"10.1.0") { 
-    $HashSHA256 = "8C9C1DFB454A8F56B76CBFB0E1071FAD03D908560CFF05E1DE462825E83DB1A4"
-    $Uri = "https://github.com/xmrig/xmrig-nvidia/releases/download/v2.14.5/xmrig-nvidia-2.14.5-cuda10-win64.zip"
- }
-else { 
-    $HashSHA256 = "9450E5483CBBDBEBDB706087E5B468FB716028751667C3F663E07395DCBC0476"
-    $Uri = "https://github.com/xmrig/xmrig-nvidia/releases/download/v2.14.5/xmrig-nvidia-2.14.5-cuda10_1-win64.zip"
+ 
+if { 
```

Comment:
Mistake?

Reply:
not really your so far behide with xmrig it needs to be re written..

Reply:
update your algos ie randomx randomsfx zergpool  im not going to do it for you

Reply:
i guess so

Reply:
cant be fucked to be honest

Reply:
algos need updating

Reply:
fixed the cuda warning Mistake. 
new xmrig is cuda 10.1 only xmrig.ps1 needs to be re written to fix --algo selection and api is not longer working. this is beyond my capabilities

Reply:
Maybe leave this one out for now...? 

Can always tackle this binary on a separate pull request...

Are all the other files complete?

Reply:
done, all complete

Reply:
running through test rigs atm all good so far,
need to add Verushash algo support ZergPool CPU (most profitable atm)
i'll do it in a couple hours